test/unit/descriptors.spec.ts

```diff
@@ -0,0 +1,97 @@
+import * as util from "../util";
+
+const trueFalseUndefTests = [[true], [false], [undefined]] as const;
+
+describe("Object.defineProperty", () => {
```

Comment:
And that one in `builtins/object.spec.ts`

Reply:
Also would be nice to have some tests for `getOwnPropertyDescriptor`/`getOwnPropertyDescriptors`